plugin/storage/cassandra/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cassandra:3.11
+FROM cassandra:4.0
```

Comment:
This is done because cassandra v4.0 image contains cqlsh 6.0.0 which is backward compatible with cassandra v3.x (https://pypi.org/project/cqlsh/)